net/unbound/files/unbound.sh

```diff
@@ -1143,9 +1143,9 @@ unbound_hostname() {
 
     *)
       # likely transparent domain with fordward-zone: clause to next router
-      echo "  domain-insecure: $UNBOUND_TXT_DOMAIN"
-      echo "  private-domain: $UNBOUND_TXT_DOMAIN"
-      echo "  local-zone: $UNBOUND_TXT_DOMAIN. $UNBOUND_D_DOMAIN_TYPE"
+      echo "  domain-insecure: \"$UNBOUND_TXT_DOMAIN\""
+      echo "  private-domain: \"$UNBOUND_TXT_DOMAIN\""
+      echo "  local-zone: \"$UNBOUND_TXT_DOMAIN.\" $UNBOUND_D_DOMAIN_TYPE"
```

Comment:
The user domain should probably be sanitized when UCI is read, but passing such check it doesn't need quotes.